api/cpp/cmake/SlintMacro.cmake

```diff
@@ -1,22 +1,37 @@
 # Copyright © SixtyFPS GmbH <info@slint-ui.com>
 # SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
 
+# Set up machinery to handle SLINT_EMBED_RESOURCES target property
+set(DEFAULT_SLINT_EMBED_RESOURCES OFF CACHE BOOL
```

Comment:
I think it makes sense for us to have a global default on the CMake side, but I think the name should be the same as the name of the target property. In CMake the global variant of these pairs have a `CMAKE_` prefix, but they can also afford to have unprefixed target properties. So I think we should always have the prefix. I can't find any precedence for `DEFAULT_` as prefix. Is there a pattern that I'm missing?

Reply:
Thanks for the pointer, the recommendation makes sense. I feel a `SLINT_` prefix would be better (`SLINT_DEFAULT_EMBED_RESOURCES`) but I can also see how that might make neck-hairs go up - I don't feel too strong about it and would be okay with your suggestion.